LetsDoStuff.WebApi/LetsDoStuff.WebApi.csproj

```diff
@@ -14,6 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.10" />
```

Comment:
Not sure it is necessary

Reply:
Ok, but what is the way to avoid circular references you are suggesting?

Reply:
Use the mapper? I'm using the AutoMapper. If you about this ask.

Reply:
Using a single model.